implementations/rust/ockam/ockam_command/src/identity/mod.rs

```diff
@@ -86,13 +87,36 @@ pub fn identity_name_parser(identity_name: &str) -> Result<String> {
 
 pub fn create_default_identity(identity_name: &str) -> String {
     let config = OckamConfig::load().expect("Failed to load config");
-    let args = GlobalArgs {
-        quiet: true,
-        ..Default::default()
-    };
-    let global_opts = CommandGlobalOpts::new(args, config);
+    let opts = CommandGlobalOpts::new(GlobalArgs::parse_from_input(), config.clone());
+    let quiet_opts = CommandGlobalOpts::new(
+        GlobalArgs {
+            quiet: true,
+            ..Default::default()
+        },
+        config,
+    );
+
+    let _ = opts
+        .shell
+        .clone()
+        .stdout()
+        .plain(format!(
+            "{} No default identity found. Creating one...",
+            "!".light_green(),
```

Comment:
We should merge https://github.com/build-trust/ockam/pull/4652 first so we can hide the `"!".light_green(),` pieces of the output.